components/dataPrivacy/HeroSection.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled         from "styled-components";
```

Comment:
Remove spaces. The linter should be picking up on this I think

Reply:
@Soreg This is so weird, I was removing some spaces in several files already, must have missed a few